service/step_defs_test.go

```diff
@@ -1862,6 +1863,69 @@ func (f *feature) aIsilonServiceWithParams(user, mode string) error {
 	return nil
 }
 
+func (f *feature) aIsilonservicewithIsiAuthTypeasfalse() error {
```

Comment:
is this still a valid test ?

Reply:
The test is valid but the function name needed an update